src/Glue.elm

```diff
@@ -123,10 +126,11 @@ poly :
     , update : subMsg -> subModel -> ( subModel, Cmd msg )
     , subscriptions : subModel -> Sub msg
     }
-    -> Glue model subModel msg subMsg
+    -> Glue model subModel msg subMsg msg
 poly { get, set, init, update, subscriptions } =
     Glue
-        { get = get
+        { msg = identity
```

Comment:
If you're not familiar with `identity` it's [single most important function in mathematics](https://en.wikipedia.org/wiki/Identity_function). It's defined in `[Basics](http://package.elm-lang.org/packages/elm-lang/core/5.1.1/Basics#identity)` and exposed to every module.